src/api/operator/utils.ts

```diff
@@ -0,0 +1,54 @@
+// Util functions that only pertain to/deal with operator API related stuff
+
+import BigNumber from 'bignumber.js'
+
+import { ONE_BIG_NUMBER } from 'const'
+
+import { RawOrder } from './types'
+
+export type OrderStatus = 'open' | 'filled' | 'expired' | 'partially filled'
+
+// TODO: move to const or app settings?
+const FILLED_ORDER_EPSILON = '0.0001' // == 0.01%
+
+function isOrderFilled(order: RawOrder): boolean {
+  let amount, executedAmount
+
+  if (order.kind === 'buy') {
+    amount = new BigNumber(order.buyAmount)
```

Comment:
question: I wonder if `BigNumber.from(...)` is faster :thinking:

Reply:
BigNumber does not have a static `.from()` method as far as I can tell https://mikemcl.github.io/bignumber.js/#bignumber